app/templates/client/app/app(stylus).styl

```diff
@@ -45,7 +45,6 @@
 <% } %>
 // Component styles are injected through grunt
 // injector
-@import "account/login/login"
```

Comment:
Oh you're right, curious, it *used* to exist. Anyway, if you're going to remove this, it should be in a separate commit, and you should do the same change for the SCSS and LESS files.